pkg/subctl/cmd/gather/types.go

```diff
@@ -24,11 +24,11 @@ import (
 )
 
 type Info struct {
+	ClientSet   kubernetes.Interface
+	DynClient   dynamic.Interface
```

Comment:
You essentially didn't change anything here so please put the fields back where they were to avoid a diff.